frontend/src/app.postcss

```diff
@@ -1,3 +1,6 @@
+/* Write your global styles here, in PostCSS syntax */
+
+@tailwind base;
```

Comment:
What's going on here?

Reply:
I think this is supposed to be postcss, I re-ran the setup for tailwind. Also updated import in main app component.